### PR TITLE
srcpkgs: remove pullmoll from maintainers field. 😢

### PR DESCRIPTION
**Pull request by Gottox**
Set them to Orphaned for now.

If you'd like to pick up a package, feel free.

### CODE REVIEW

**Comment by ericonr** [comment]
How would you like to do it for adopting packages? Can I push to master and you rebase the PR, or would you rather I list them here, or just push after this is merged?

**Comment by Gottox** [comment]
just suggest the change in the PR. I think that's the easiest way.

**Comment by ericonr** [comment]
Maintainer field `Érico Nogueira <ericonr@disroot.org>`: aisleriot freerouting json-c++ qbittorrent wildmidi

**Comment by Gottox** [comment]
Could you do an suggestion as described [here?](https://haacked.com/archive/2019/06/03/suggested-changes/)

File: srcpkgs/aisleriot/template [hunk 1]
```diff
@@ -11,7 +11,7 @@ hostmakedepends="desktop-file-utils glib-devel guile intltool itstool pkg-config
 makedepends="guile-devel libcanberra-devel librsvg-devel libatomic_ops-devel"
 depends="guile yelp"
 short_desc="GNOME solitaire card game"
-maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (ericonr) [inline srcpkgs/aisleriot/template:14]:
```suggestion
maintainer="Érico Nogueira <ericonr@disroot.org>"
```

File: srcpkgs/freerouting/template [hunk 2]
```diff
@@ -5,7 +5,7 @@ revision=2
 hostmakedepends="gradle openjdk11"
 depends="openjdk11"
 short_desc="Advanced PCB autorouter"
-maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (ericonr) [inline srcpkgs/freerouting/template:8]:
```suggestion
maintainer="Érico Nogueira <ericonr@disroot.org>"
```

File: srcpkgs/json-c++/template [hunk 3]
```diff
@@ -7,7 +7,7 @@ build_style=cmake
 configure_args="-DJSON_MultipleHeaders=ON"
 checkdepends="git"
 short_desc="JSON for modern C++"
-maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (ericonr) [inline srcpkgs/json-c++/template:10]:
```suggestion
maintainer="Érico Nogueira <ericonr@disroot.org>"
```

File: srcpkgs/qbittorrent/template [hunk 4]
```diff
@@ -11,7 +11,7 @@ hostmakedepends="automake libtool pkg-config qt5-host-tools qt5-qmake qt5-tools"
 makedepends="libtorrent-rasterbar-devel qt5-declarative-devel qt5-svg-devel"
 depends="qt5-svg"
 short_desc="Free software alternative to µtorrent"
-maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (ericonr) [inline srcpkgs/qbittorrent/template:14]:
```suggestion
maintainer="Érico Nogueira <ericonr@disroot.org>"
```

File: srcpkgs/wildmidi/template [hunk 5]
```diff
@@ -9,7 +9,7 @@ hostmakedepends="pkg-config"
 makedepends="alsa-lib-devel libopenal-devel"
 depends="libwildmidi"
 short_desc="Simple software midi player and core softsynth library"
-maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```
Comment (ericonr) [inline srcpkgs/wildmidi/template:12]:
```suggestion
maintainer="Érico Nogueira <ericonr@disroot.org>"
```

**Comment by Chocimier** [comment]
Adopting qtcreator in separate PR.

**Pull request merged**